pegasus/sites.v3/code.org/public/css/ai-pl-101-styles.scss

```diff
@@ -180,3 +181,43 @@
   padding-bottom: calc(64px - 10px);
 }
 
+.img {
+  &--max {
+    max-width: 100%;
+  }
+}
+
+.video-101 {
+  padding-bottom: 8px;
+
+  &__text {
+    margin-bottom: 32px;
+  }
+
+  & a.link-button {
```

Comment:
@dmantonyuk I didn't catch this before, but you can add [this existing `secondary` class](https://github.com/code-dot-org/code-dot-org/blob/34175761b482c6a4df605c1823389a04e431d79c/pegasus/sites.v3/code.org/public/css/design-system-pegasus.scss#L226-L233) to these buttons in the haml file and don't need to duplicate the styles here.